### PR TITLE
DDP-6047 rarex: quality-of-life edits

### PR DESCRIPTION
**Pull request by yufengwng**
## Context

See DDP-6047. I had to make changes so that we have 3 different activities for quality-of-life.

## Checklist

- [x] I have labeled the type of changes involved using the `C-*` labels.
- [x] I have assessed potential risks and labeled using the `R-*` labels.
- [x] I have considered error handling and alerts, and added `L-*` labels as needed.
- [x] I have considered security and privacy, and added `I-*` labels as needed
- [x] I have analyzed my changes for stability, fault tolerance, graceful degradation, performance bottlenecks and written a brief summary in this PR.

## FUD Score

- [x] :relaxed: All good, business as usual!

## How do we demo these changes?

Need to redeploy configs to environment.

## Testing

- [x] I have written zero automated tests but have poked around locally to verify proper functionality

## Release

Need to redeploy configs to environment.

### CODE REVIEW

File: study-builder/studies/rarex/i18n/en.json
Comment (yufengwng) [inline study-builder/studies/rarex/i18n/en.json:568]: Did 3 things here:
* renamed `qol_guardian_*` -> `qol_patient_*`
* renamed `qol_parent_*` -> `qol_child_*`
* added `qol_child_instruction`

File: study-builder/studies/rarex/quality-of-life-parental.conf
Comment (yufengwng) [inline study-builder/studies/rarex/quality-of-life-parental.conf:4]: This is split into `quality-of-life-child.conf` and `quality-of-life-patient.conf`.

File: study-builder/studies/rarex/snippets/picklist-question-difficulty-child.conf
Comment (yufengwng) [inline study-builder/studies/rarex/snippets/picklist-question-difficulty-child.conf:96]: Created a copy of file to use `qol_unable_to_answer_due_to_child`.

File: study-builder/studies/rarex/snippets/picklist-question-how-often-child.conf
Comment (yufengwng) [inline study-builder/studies/rarex/snippets/picklist-question-how-often-child.conf:115]: Created a copy so to make it use `qol_unable_to_answer_due_to_child`.

File: study-builder/studies/rarex/study.conf
Comment (yufengwng) [inline study-builder/studies/rarex/study.conf:1052]: We likely need to do this by "age in locale" (what I have now) rather than hardcoded `>= 18`. Still waiting on final decision here.

File: study-builder/studies/rarex/study.conf
Comment (yufengwng) [inline study-builder/studies/rarex/study.conf:1282]: `420 sec / 60` is 7 minutes. What we want is 7 days (aka 604800 secs). This is likely a leftover from local testing.

File: study-builder/studies/rarex/study.conf
Comment (yufengwng) [inline study-builder/studies/rarex/study.conf:1945]: Oddly enough, the DOB question in the self-consent is not required. Don't know why. Can fix it in another PR.

File: study-builder/studies/rarex/study.conf
Comment (yufengwng) [inline study-builder/studies/rarex/study.conf:2487]: I think we need these events. Not too sure. Adding them anyways.

File: study-builder/studies/rarex/study.conf
Comment (yufengwng) [reply study-builder/studies/rarex/study.conf:1052]: [Confirmed](https://broadinstitute.slack.com/archives/C01A6HWK3PX/p1620740408140600?thread_ts=1620671795.091400&cid=C01A6HWK3PX)! We want "age in locale" rules here, so this should be good-to-go.

**Comment by yufengwng** [comment]
Not sure why my "consent dob edits" PR was merged into this PR already. But oh well, there it is.

File: study-builder/studies/rarex/study.conf
Comment (yufengwng) [reply study-builder/studies/rarex/study.conf:1945]: Should be fixed now.

**Comment by YuriyPavlovBI** [comment]
Looks good

**Review by YuriyPavlovBI — APPROVED** [review]

File: study-builder/studies/rarex/i18n/en.json
Comment (MocanaAtBroad) [inline study-builder/studies/rarex/i18n/en.json:598]: This pretty specific, but in DDP-5726 I am seeing the text to simply be: Unable to answer . We sure about this one?

Comment (yufengwng) [reply study-builder/studies/rarex/i18n/en.json:598]: See DDP-6047.

> For the Quality of Life <18: for all questions related to child (start with "My child…"), change response "Unable to answer" to "Unable to answer because of my child's condition or communication skills."

File: study-builder/studies/rarex/quality-of-life-child.conf
Comment (MocanaAtBroad) [inline study-builder/studies/rarex/quality-of-life-child.conf:610]: This section missing title (Peer relationships)

Comment (yufengwng) [reply study-builder/studies/rarex/quality-of-life-child.conf:610]: Thanks! Will fix.

**Pull request merged**
